samples/quickperf/src/main/java/com/google/cloud/jdbc/quickperf/QuickPerf.java

```diff
@@ -0,0 +1,497 @@
+import java.sql.Statement;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.cli.*;
+import org.apache.commons.lang3.ArrayUtils;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.google.cloud.jdbc.quickperf.config.Config;
+import com.google.cloud.jdbc.quickperf.config.ConfigParser;
+import com.google.cloud.jdbc.quickperf.config.QueryParam;
+import com.google.cloud.spanner.Dialect;
+import com.google.cloud.spanner.jdbc.CloudSpannerJdbcConnection;
+
+import net.datafaker.Faker;
+
+public class QuickPerf extends Thread {
+    private static String BREAK_STR = "###################################################################################################";
+    private static String DEFAULT_TAG = "perftest_" + (new Random()).nextInt(300);
+
+    // TODO: make measurementfile configurable
+    private static String MEASURES_FILE_NAME = "measures.txt";
+
+    // perf measurement
+    private float measures[];
+
+    // configs
+    Config config;
+
+    private List<String> sampledValueList = new ArrayList<String>();
+
+    private int progress;
+
+    public static void main(String[] args)
+            throws SQLException, InterruptedException, StreamReadException, DatabindException, IOException {
+        Options options = new Options();
+
+        options.addOption(QuickPerf.addOption("c", "config", true, "Config File"));
+
+        CommandLineParser parser = new DefaultParser();
+        HelpFormatter formatter = new HelpFormatter();
+        CommandLine cmd = null;
+
+        ZonedDateTime testStartTimestamp = ZonedDateTime.now();
+
+        try {
+            cmd = parser.parse(options, args);
+        } catch (ParseException e) {
+            System.out.println(e.getMessage());
+            formatter.printHelp("utility-name", options);
+
+            System.exit(1);
+        }
+
+        // System.out.println("#### " + cmd.getOptionValue("config"));
+
+        Config config = ConfigParser.parseConfigFile(cmd.getOptionValue("config"));
+
+        float measures[] = new float[config.getIterations() * config.getThreads()];
+
+        List<QuickPerf> threadList = new ArrayList<QuickPerf>();
+        for (int i = 0; i < config.getThreads(); i++) {
+            QuickPerf thread = new QuickPerf(config);
+            if (config.getSamplingQuery() != null) {
+                thread.runSampling();
+            }
+            thread.start();
+            threadList.add(thread);
+        }
+
+        // ProgressBar Tracker Thread
+        ProgressTracker progressTracker = progressTracker = new ProgressTracker(threadList,
+                config.getIterations() * config.getThreads());
+
+        progressTracker.start();
+
+        progressTracker.join();
```

Comment:
nit: remove the redundant spaces between these lines of code

Reply:
done